Updates to SSL-related content (#594)
dnsimple/dnsimple-support: content/articles/what-is-ssl-certificate-chain.markdown

    If your server isn't on the wizard, you can still obtain the proper files through it, then follow your web server's documentation to determine how to properly install your domain certificate and intermediate certificates.

1.  #### What happens if I don't install an Intermediate SSL certificate?

    If you don't install one or more intermediate SSL certificate, you break the certificate chain. That means you create a gap between a specific (end-user or intermediate) certificate and its issuer. When a device can't find a trusted issuer for a certificate, the certificate and the entire chain, from the intermediate certificate down to the final cerficate, can't be trusted.

    As a result, your final certificate won't be trusted. Web browsers will display an "Invalid certificate" or "certificate not trusted" error.

1.  #### How can I shorten the SSL certificate chain in my browser?

    This isn't possible. The only way to shorten a chain is to promote an intermediate certificate to root. Ideally, you should promote the certificate that represents your Certificate Authority, that way the chain will consist of just two certificates.

>     Root certificates are packaged with the browser software. The list can't be altered if not from the browser maintainers.

Reviewer: This sentence feels weird to me. Does it make sense to change it to 

```suggestion
    Root certificates are packaged with the browser software. The list can only be altered by the browser maintainers.
```

?
